bokeh/util/serialization.py

```diff
@@ -150,6 +150,10 @@ def transform_array_to_list(array):
         transformed[np.isposinf(array)] = 'Infinity'
         transformed[np.isneginf(array)] = '-Infinity'
         return transformed.tolist()
+    elif (array.dtype.kind == 'O' and pd.isnull(array).any()):
```

Comment:
It it better to look for matches to np.nan? That seems dicey. And np.isnan doesn't work